docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -1569,6 +1398,8 @@ kubectl --context $CLUSTER_SURVIVING port-forward services/$HELM_RELEASE_NAME-ze
 zbctl status --insecure --address localhost:26500
 ```
 
+<!-- HELP NEEDED !-->
```

Comment:
Line 1398 includes zbctl, but I'm not sure how to amend it in the context of kubectl. @conceptualshark can you suggest someone to provide some help here? How can we use the C8 REST API topology endpoint in this context?

Reply:
@conceptualshark ignore this, I realized I didn't remove all the zbctl specific instructions from this page!